doc/contributer_guide.rst

```diff
@@ -0,0 +1,41 @@
+.. _contributer_guide:
+
+Contributer Guide
```

Comment:
It's spelled **Contributor** 

Could you change throughout?

Reply:
oh sorry. of course